recipe/0002-win-gperf-lf.patch

```diff
@@ -0,0 +1,63 @@
+--- Source/WebCore/platform/network/create-http-header-name-table.orig	2020-11-24 14:11:54.102671419 +1000
```

Comment:
Is is possible to make `m2-gperf` understand Windows line endings? Or do we need a MSVC version of `gperf`? Would be nice to remove these patches.

Reply:
I actually looked into this and appears that `gperf` always assumes Unix line endings (see `getline.cc`). Not sure much can be done without major surgery on the sources. Will keep the patches now now...